snap/snapcraft.yaml

```diff
@@ -171,12 +210,44 @@ parts:
       - glib-networking
       - libglib2.0-bin
       - libglib2.0-0
+      - libpci3
+      - libprotobuf-c1
+      - libftdi1-2
+      - libmm-glib0
+      - libqmi-glib5
+      - libmbim-glib4
+      # pkttyagent
+      - polkitd
     prime:
       # we explicitly don't want /usr/bin/gpgconf
       # this will cause gpgme to error finding it
       # but that also avoids trying to use non-existent
       # /usr/bin/gpg2
-      - -usr/bin
+      - -usr/bin/dirmngr
```

Comment:
why not take out of all `/usr/bin` like before?  I figured by putting the snap into a different prefix (`/`) this is a sane way to go.

Reply:
Done.